src/CONST.ts

```diff
@@ -52,6 +52,8 @@ const CONST = {
         MIN_SIZE: 240,
     },
 
+    API_RECEIPT_EXTENSIONS: ['jpg', 'jpeg', 'gif', 'png', 'pdf', 'htm', 'html', 'text', 'rtf', 'doc', 'tif', 'tiff', 'msword', 'zip', 'xml', 'message'],
```

Comment:
Isn't it better to move this to `CONST.API_ATTACHMENT_VALIDATIONS`? And name it to `ALLOWED_RECEIPT_EXTENSIONS`

Reply:
Sounds good. Updated!